docs/site/home.html

```diff
@@ -96,26 +96,35 @@ <h3 class="block-title">What is it?</h3>
         </div>
 
         <div hljs language="js" ng-show="selected == 'compute engine'">
-// Auth is handled automatically inside GCE.
-var gcloud = require('gcloud');
-var dataset = gcloud.datastore.dataset();
-var productKey = dataset.key(['Product', 123]);
+// Authentication is automatic
+// inside Google Compute Engine.
+var gcloud = require('gcloud')({
+  projectId: 'grape-spaceship-123'
+});
+
+var gcs = gcloud.storage();
 
-dataset.get(productKey, function(err, entity) {
-  console.log(err, entity);
+var backups = gcs.bucket('backups');
+backups.upload('db.zip', function(err, file) {
```

Comment:
It didn't confuse me but I didn't consider if it made much sense. Thinking now, the file that is returned already has its metadata so getting metadata again actually doesn't make sense. Also creating a read stream doesn't make sense either because it would be smarter to create a read stream from the local file, not the remote one. 

The best idea I think is that once I upload a file to storage, I want to store its media url for download or embedding (e.g. in an image tag) somewhere, and we can get that from the metadata (technically its a well-known, but we should verify that it uploaded properly first).

So something like this I think would be useful:

``` js
var assets = gcs.bucket('my-app-assets');
assets.upload('kitten.jpg', function(err, file) {
  if (!err) {
    // file.metadata.mediaLink
  }
});
```